djangoproject/templates/aggregator/ecosystem.html

```diff
@@ -0,0 +1,160 @@
+      &mdash; Django Conference Europe YouTube channel
+    </li>
+    <li>
+      <a href="https://djangotv.com">Django.tv</a>
+      &mdash; Django video content platform
+    </li>
+    <li>(official Django social accounts)</li>
+  </ul>
+
+
+  <p>Explore! A whole world awaits you.</p>
+
+
+  <h2>Packages</h2>
+
+  <p>We could never list everything but here are some packages that the Steering Council all agree are
+    incredibly useful and well supported. We use many of these in all of our projects.</p>
+
+  <p>We will refresh this list at least yearly to keep up to date.</p>
+
+  <h3>Debugging & Development Tools</h3>
+  <ul>
+    <li><a href="https://github.com/jazzband/django-debug-toolbar">Django Debug Toolbar</a> &mdash; A configurable set of panels that display various debug information about the current request/response.</li>
+    <li><a href="https://github.com/django-browser-reload/django-browser-reload">django-browser-reload</a> &mdash; Enables automatic browser reloading during development when code changes are detected.</li>
+    <li><a href="https://github.com/django-extensions/django-extensions">Django Extensions</a> &mdash; A collection of custom extensions for Django, including management commands, model field types, and more.</li>
+  </ul>
+
+  <h3>Storage & Static Files</h3>
+  <ul>
+    <li><a href="https://github.com/jschneier/django-storages">django-storages</a> &mdash; Provides a variety of storage backends for Django, including S3, Azure, and more.</li>
+    <li><a href="https://github.com/evansd/whitenoise">Whitenoise</a> &mdash; Serves static files directly from Django without the need for a separate web server.</li>
+  </ul>
+
+  <h3>API Development</h3>
+  <ul>
+    <li><a href="https://github.com/encode/django-rest-framework">Django REST Framework (DRF)</a> &mdash; A powerful and flexible toolkit for building Web APIs in Django.</li>
+    <li><a href="https://github.com/vitalik/django-ninja">Django Ninja</a> &mdash; A fast, asynchronous-ready web framework for building APIs with Django and Python type hints.</li>
+  </ul>
+
+  <h3>Content Management Systems (CMS)</h3>
+  <ul>
+    <li><a href="https://github.com/wagtail/wagtail">Wagtail</a> &mdash; An open-source Django CMS focused on flexibility and user experience.</li>
+    <li><a href="https://github.com/django-cms/django-cms">Django CMS</a> &mdash; A user-friendly and developer-friendly CMS for Django, offering multilingual support and a plugin architecture.</li>
+    <li><a href="https://github.com/feincms/feincms">FeinCMS</a> &mdash; A Django-based CMS with a focus on extensibility and minimalism.</li>
+  </ul>
+
+  <h3>Authentication & Authorization</h3>
+  <ul>
+    <li><a href="https://github.com/pennersr/django-allauth">Django Allauth</a> &mdash; Integrated set of Django applications addressing authentication, registration, account management, and third-party (social) account authentication.</li>
```

Comment:
```suggestion
    <li><a href="https://allauth.org/">Django Allauth</a> &mdash; Integrated set of Django applications addressing authentication, registration, account management, and third-party (social) account authentication.</li>
```
We should consider using the custom domains when possible, especially on individual's repos in case they want to move off GitHub. Feel free to dismiss these comments.